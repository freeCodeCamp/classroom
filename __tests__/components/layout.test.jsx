import Layout from '../../components/layout';
import React from 'react';
import renderer from 'react-test-renderer';


describe('Layout', () => {
  it('displays expected children', () => {
    const tree = renderer
      .create(
        <Layout>
          <h1>I love freecodecamp</h1>
        </Layout>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
