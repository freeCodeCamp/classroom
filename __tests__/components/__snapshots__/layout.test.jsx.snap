// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout displays expected children 1`] = `
<div>
  <h1>
    I love freecodecamp
  </h1>
</div>
`;
